user snippets/addDaysToToday.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Run a small snippet of JavaScript during a mabl flow/journey
+ *
+ * @param {object} mablInputs - Object containing mabl inputs such as variables (mablInputs.variables).
+ *                              Use mablInputs.variables.user for user defined variables
+ *                              (For example myVar may be accessed as mablInputs.variables.user.myVar)
+ *
+ * @param {function} callback - A callback function that must be called to complete
+ *                              the javascript step and provide a value to the following
+ *                              steps of the flow/journey. A return statement from this
+ *                              function call will not provide any results for use
+ *                              in the following steps in this flow or journey.
+ */
+ function mablJavaScriptStep(mablInputs, callback) {
+
+  // addDays変数の数だけ日付を進める, Advance the date by the number of addDays variables
+  let addDays = mablInputs.variables.user.addDays;
```

Comment:
Can you use "var" rather than "let" so that this will work on ES5 JavaScript too, like Internet Explorer?